src/ops/ops.go

```diff
@@ -259,3 +260,24 @@ func (o *ops) RemovePV(pvName string) error {
 	}
 	return err
 }
+
+func (o *ops) GetMCSLogs() (string, error) {
```

Comment:
i think you should add a comment  that explains what we talked about: why we look only at first file, why it is enough etc'

Reply:
ok